lmdeploy/model.py

```diff
@@ -720,7 +720,7 @@ def get_prompt(self, prompt, sequence_start=True):
 @MODELS.register_module(name='chatglm2-6b')
 class ChatGLM2(BaseModel):
 
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
Do we need add kwargs in falcon?

Reply:
it is fixed in https://github.com/InternLM/lmdeploy/pull/1168. maybe should remove it and wait for the pr

Reply:
#1168 has been merged